Sources/Basic/PathShims.swift

```diff
@@ -17,22 +17,11 @@
  while making it fairly easy to find those calls later.
 */
 
-import SPMLibc
 import Foundation
 
 /// Returns the "real path" corresponding to `path` by resolving any symbolic links.
 public func resolveSymlinks(_ path: AbsolutePath) -> AbsolutePath {
-    let pathStr = path.pathString
-
-    // FIXME: We can't use FileManager's destinationOfSymbolicLink because
-    // that implements readlink and not realpath.
-    if let resultPtr = SPMLibc.realpath(pathStr, nil) {
-        let result = String(cString: resultPtr)
-        // FIXME: We should measure if it's really more efficient to compare the strings first.
-        return result == pathStr ? path : AbsolutePath(result)
-    }
-
-    return path
+    return AbsolutePath(path.pathString.standardizingPath)
```

Comment:
I am really scared with using this method. It goes through a bunch of weird code path in foundation. It might be best to conditionally keep the above realpath implementation.

Reply:
BTW, the comment doesn't make sense to me - `realpath` will canonicalize the path *AND* go through `readlink`.  It effectively is just `FileManager.default.destinationOfSymbolicLink(atPath:).standardizingPath`.  http://man7.org/linux/man-pages/man3/realpath.3.html

Reply:
destinationOfSymbolicLink wasn’t converting /var/blah to /private/var/blah which ended up in some failures

Reply:
Right, that would require going through `.standardizingPath` I believe.